test_improve_iti.jl

```diff
@@ -0,0 +1,87 @@
+import Dolo
```

Comment:
Do we ever intend to have this test be run by CI? I don't think it is right now, and that's ok with me.